src/base/emu-i386/simx86/protmode.h

```diff
@@ -42,6 +42,10 @@ typedef struct {
 	unsigned short Oldsel, Attrib __attribute__ ((packed));
```

Comment:
But I simply don't see
where you actually remove
Attrib. Its still here.
So what's changed?